reactDashboard/src/components/DataDefinitions/DataDefinitions.css

```diff
@@ -58,10 +71,28 @@
   font-weight: bold;
 }
 
+.darkmode .data-definitions .defined-term,
+.darkmode .data-definitions .definitions-heading {
+  color: #c1def1;
+}
+
+.darkmode .data-definitions hr {
+  border: 1px solid #cccccc;
+}
+
 .data-definitions-body .source-attribution {
   color: #2a84bb;
 }
 
-.data-definitions .definition-paragraph {
-  padding-bottom: 20px !important;
+.darkmode .data-definitions .card-body {
+  background-color: #262626;
+  color: #f2f2f2;
+}
+
+.card {
```

Comment:
Thats a very generic class to be overriding - are you sure there are no side effects? Could it be prefixed with .data-definitions?

Reply:
✅